modules/tinymce/src/themes/silver/test/ts/browser/editor/SilverFixedToolbarContainerPriorityTest.ts

```diff
@@ -0,0 +1,125 @@
+﻿import { ApproxStructure, Assertions } from '@ephox/agar';
+import { after, before, describe, it } from '@ephox/bedrock-client';
+import { TinyHooks } from '@ephox/mcagar';
+import { Insert, Remove, SugarBody, SugarElement } from '@ephox/sugar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.themes.silver.editor.SilverFixedToolbarContainerTest', () => {
+  const toolbar: SugarElement<HTMLDivElement> = SugarElement.fromHtml('<div style="margin: 50px 0;"></div>');
+  const toolbarWithId: SugarElement<HTMLDivElement> = SugarElement.fromHtml('<div id="toolbar" style="margin: 50px 0;"></div>');
+  before(() => {
+    Insert.append(SugarBody.body(), toolbar);
+    Insert.append(SugarBody.body(), toolbarWithId);
+  });
+
+  after(() => {
+    Remove.remove(toolbar);
+  });
+
+  const hook = TinyHooks.bddSetup<Editor>({
+    inline: true,
+    fixed_toolbar_container_target: toolbar.dom,
+    fixed_toolbar_container: '#toolbar',
+    menubar: 'file',
+    toolbar: 'undo bold',
+    base_url: '/project/tinymce/js/tinymce'
+  }, [ Theme ]);
+
+  it('Check basic structure', () => {
```

Comment:
This test can be a lot simpler because we only need to check that the toolbar exists, something like:

```
    const editor = hook.editor();
    await TinyUiActions.pWaitForUi(editor, '#toolbar .tox-editor-header');
    Assertions.assertPresence('Check that only one toolbar exists', { '.tox-editor-header': 1 }, SugarBody.body());
```

`TinyUiActions` is from `@ephox/mcagar`
`Assertions` is from `@ephox/agar`

Reply:
Awesome, thanks for the example.
The tests have been adapted.